react-native-cli/index.js

```diff
@@ -42,10 +43,11 @@ if (cli) {
   switch (args[0]) {
   case 'init':
     if (args[1]) {
-      init(args[1]);
+      var verbose = process.argv.indexOf('--verbose') >= 0;
+      init(args[1], verbose);
     } else {
       console.error(
-        'Usage: react-native init <ProjectName>'
+        'Usage: react-native init <ProjectName> [--verbose]'
       );
       process.exit(1);
     }
```

Comment:
The goal of this file is to be dead simple and just forward everything to the local cli. The only thing it handles is init which bootstraps the project.

I don't think that we want to use an option parser here